packages/TEMPLATE/uninstall

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# To use if root is required
+[ "$UID" -eq 0 ] || (echo "installation requires root access"; exec sudo "$0" "$@")
+
+# Only uninstall general linux dependencies in rare circumstances -- if another package requires it, there is risk of breaking their dependency.
+
+# Remove general setup / configuration
+
+# Recipe for singularity images
+rm ~/.planutils/bin/images/<image name>
+
+# No need to uninstall planutils dependencies
+# No need to revert the "run" script if it exists
```

Comment:
Again, I don't understand why the run script has to be moved/modified at all.

Reply:
As far as I can tell, it's the only way for `downward` on the command line anywhere to invoke the `downward/run` script. No?

Reply:
Doing a re-direct...should work now through the single generated bash script with one extra level of conditional checking.